common/datasets/librispeech.py

```diff
@@ -255,14 +266,153 @@ def get_ogg_zip_dict(create_alias_with_prefix=None):
     return ogg_zip_dict
 
 
-def export_all_datasets(path_prefix):
+def get_arpa_lm_dict(create_alias_with_path_prefix=None):
+    """
+    Download the ARPA language models from OpenSLR,
+    valid keys are: "3gram" and "4gram".
+
+    :param create_alias_with_path_prefix:
```

Comment:
I don't find it completely intuitive from the name what this argument does and what I should pass for it, so you might consider describing it here.